test/scripts/boot-image

```diff
@@ -379,7 +379,7 @@ def main():
     # Keep test/scripts/imagetestlib.py:CAN_BOOT_TEST in sync as it
     # has the same checks again
     match image_type:
-        case "qcow2" | "server-qcow2":
+        case "qcow2" | "generic-qcow2":
```

Comment:
This rename should probably happen before https://github.com/osbuild/images/pull/1973/commits/6ac0d37bdd1f407c0bf047f6205af5b5e2db8130, i.e. when all image types were renamed to `generic-*`.

However, related to the changes in https://github.com/osbuild/images/pull/1973/commits/6ac0d37bdd1f407c0bf047f6205af5b5e2db8130, why not just test all three `qcow2` image types? IOW `"qcow2" | "generic-qcow2" | "server-qcow2"`